opensubtitles/find.go

```diff
@@ -0,0 +1,120 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+type FindService service
+
+type FeatureOptions struct {
```

Comment:
Codacy found an issue: [exported type FeatureOptions should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)